File: 백준/Bronze/2562. 최댓값/최댓값.js
const input = require("fs").readFileSync(0).toString().split("\n").map(a => +a);

let max = 0;
let index = 0;
for (let i = 0; i < 9; i++) {
    if (input[i] > max) {
        max = input[i];
    }
}
for (let i = 0; i < 9; i++) {
    index++;
    if (input[i] === max) {
        console.log(`${max}
${index}`);
        break;
    }
}

File: 백준/Bronze/2562. 최댓값/최댓값.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int max = 0;
        int index = 0;
        for (int i = 1; i <= 9; i++) {
            int n = Integer.parseInt(br.readLine());
            if (n > max) {
                max = n;
                index = i;
            }
        }
        System.out.printf("%d%n%d", max, index);
    }
}
